roman_imsim/stamp.py

```diff
@@ -311,9 +311,10 @@ def draw(self, prof, image, method, offset, config, base, logger):
                 logger.info("offset = %r", offset)
                 logger.info("wcs = %r", image.wcs)
                 raise
-            # Some pixels can end up negative from FFT numerics.  Just set them to 0.
-            fft_image.array[fft_image.array < 0] = 0.0
-            fft_image.addNoise(galsim.PoissonNoise(rng=self.rng))
+            if method == "fft_poisson":
+                # Some pixels can end up negative from FFT numerics.  Just set them to 0.
+                fft_image.array[fft_image.array < 0] = 0.0
```

Comment:
Why should this be set to 0 only for the so-called `fft_poisson` method and not `fft`?

Reply:
I think this necessary to do the poisson sampling, it was already there so I just kept it. For the classic `fft` I don't think GalSim does it so I wanted to keep the `fft` drawinf as close as possible as the default GalSim.